docs/resources/ipam_address_block.md

```diff
@@ -29,6 +29,12 @@ data "bloxone_dhcp_option_codes" "option_code" {
   }
 }
 
+data "bloxone_federation_federated_realms" "federated_realm" {
```

Comment:
It is advisable to use Resource instead of Data source as the user might not have a federation realm with the same name . 
Same for subnet and ip space.

Reply:
done